hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -967,7 +967,11 @@ public String startDeletePartitionCommit() {
    * @return the instant generated for the delete partition commit
    */
   public String startDeletePartitionCommit(HoodieTableMetaClient metaClient) {
-    return tableServiceClient.startDeletePartitionCommit(metaClient).requestedTime();
+    String instantTime = tableServiceClient.startDeletePartitionCommit(metaClient).requestedTime();
```

Comment:
how about the method in line 960, maybe we can make that method invoke this one.